specification/ai/Azure.AI.Projects/connections/model.tsp

```diff
@@ -76,6 +76,9 @@ enum AuthenticationType {
   @doc("Shared Access Signature (SAS) authentication")
   SAS: "SAS",
 
+  @doc("Custom Keys authentication")
+  customKeys: "CustomKeys",
```

Comment:
This should just be named `custom: "custom"` based on what I see in the AI Foundry project Lina shared.